### ISSUE TITLE
[Feature] Is it possible to hide unwatched episode name titles? 

### ISSUE THREAD

**Issue by jkoor** [jurialmunkey/skin.arctic.horizon.2#1011]
### Skin section

Library Views

### Feature description

In kodi, I can hide unwatched tvshows eposide plots and thumbs to avoid **spoiler**.
(`Settings/Media/Videos/Show information for unwatched items`)

But the eposide title can't be hidden, sometimes there are spoilers in the eposide title.
For example: `1x10: xxx is killed`
So is it possible to hide unwatched episode name titles?

### Why is this feature needed? Please justify and explain.

Add a option to hide unwatched episode name titles.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by Bindou01** [comment]
Good idea. In the same vein, it would also be nice to have the thumbnail blurred on the unseen episodes. And make it appear sharp, only when it's playing or read. This would avoid spoilers, and it would be extremely innovative for once. I don't know if it's possible though. (An idea to also suggest on the Fuse project)
